lib/Models/WebProcessingServiceCatalogItem.js

```diff
@@ -105,194 +99,162 @@ WebProcessingServiceCatalogItem.prototype._getValuesThatInfluenceLoad = function
     return [this.wpsResponseUrl, this.wpsResponse];
 };
 
+WebProcessingServiceCatalogItem.prototype._getGeojson = function() {
```

Comment:
If this is never meant to be called from outside this class, you can make it a simple function that takes the catalog item as its first parameter.  Generally a function on the prototype, named with an underscore, should be reserved for a "protected" method.  i.e. one that's meant to be called from base or derived classes but not from outside the class.